src/components/DocumentRenderer/WebViewFrame.tsx

```diff
@@ -64,9 +64,14 @@ export const WebViewFrame: FunctionComponent<WebViewFrame> = ({
           const templates = window.openAttestation({type: "GET_TEMPLATES", payload: documentToRender});
           window.ReactNativeWebView.postMessage(JSON.stringify(templates));
 
-          // Add spacing container to the bottom of the document so that it isn't blocked by the bottom sheet
+          // Add spacing container to ensure the document isn't blocked by the bottom sheet
           const spacingContainer = document.createElement('div');
-          spacingContainer.style.height = '30vh';
```

Comment:
We are using double quotes, I wonder why is lint not complaining....

Reply:
i think it's because it's a string lol, i'll update it